packages/EasyUtils/src/CreditCard/CreditCardNumberValidator.php

```diff
@@ -0,0 +1,148 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyUtils\CreditCard;
+
+final class CreditCardNumberValidator implements CreditCardNumberValidatorInterface
```

Comment:
Please add a link to the source of this code in the PR description. Just for other reviewers.

Reply:
Done